src/mongo/db/pipeline/document_source_skip_test.cpp

```diff
@@ -65,5 +65,28 @@ TEST_F(DocumentSourceSkipTest, ShouldPropagatePauses) {
     ASSERT_TRUE(skip->getNext().isEOF());
 }
 
+TEST_F(DocumentSourceSkipTest, SkipsChainedTogetherShouldNotOverFlowWhenOptimizing) {
+    // $skip should not optimize if overflow
+    auto skipShort = DocumentSourceSkip::create(getExpCtx(), 1);
+    auto skipLong = DocumentSourceSkip::create(getExpCtx(), std::numeric_limits<long long>::max());
+    Pipeline::SourceContainer overFlowContainer;
```

Comment:
total nitpick, but if overflow is one word, this should be 'overflowContainer' not 'overFlowContainer'.